src/Service/PlanningService.php

```diff
@@ -0,0 +1,330 @@
+<?php
+
+namespace App\Service;
+
+use App\Model\Planning\Assignee;
+use App\Model\Planning\AssigneeProject;
+use App\Model\Planning\Issue;
+use App\Model\Planning\PlanningData;
+use App\Model\Planning\Project;
+use App\Model\Planning\SprintSum;
+use App\Model\Planning\Weeks;
+use App\Repository\IssueRepository;
+use Doctrine\Common\Collections\ArrayCollection;
+
+class PlanningService
+{
+    public function __construct(
+        private readonly DateTimeHelper $dateTimeHelper,
+        private readonly IssueRepository $issueRepository,
+        protected readonly float $weekGoalLow,
+        protected readonly float $weekGoalHigh,
+    ) {
+    }
+
+    private const UNNAMED_STR = 'unnamed';
+    private const WEEKS_IN_SPRINT_PERIOD = 3;
+    private const WEEKS_IN_SUPPORT_PERIOD = 1;
+
+    /**
+     * Retrieves the planning data containing the weeks, issues, assignees, and projects.
+     *
+     * @return PlanningData The planning data object
+     */
+    public function getPlanningData(): PlanningData
+    {
+        $planning = new PlanningData();
+        $planning->weeks = $this->buildPlanningWeeks($planning);
+        $allIssues = $this->issueRepository->findAll();
```

Comment:
This should be changed to a filtered result where only issues with dueDate in the current year are included.